### ISSUE TITLE
Update README 

### ISSUE THREAD

**Issue by lukehedger** [jaakmusic/primitives#29]
- [x] 'Goal' of library
  - Reusable styled primitive components
  - HTML-based semantics
  - Unopinionated defaults (avoid hardcoded values!)
  - ...

- [x] Recommended usage/extension pattern
> Library primitives -> Application primitives -> Components (primitive composition)

- [x] How to create a theme
```
import { theme } from 'jaak-primitives'
import { ThemeProvider } from 'styled-components'

// this is optional
const customTheme = {
  palette: {
    primary: '#478134',
  }
}

<ThemeProvider theme={theme(customTheme)}>
  ...
</ThemeProvider>
```

**Comment by lukehedger** [comment]
Should also consider integration or inspiration from these libraries:
- https://github.com/Decisiv/styled-components-modifiers
- https://github.com/alanbsmith/component-library-starter

**Comment by lukehedger** [comment]
Closed by #39
